### PR TITLE
Add Quadratic Solver

### PR DESCRIPTION
**Pull request by wisentini**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by wisentini** [comment]
I don't understand what's wrong with my commit. Can someone explain it to me?

File: README.md [hunk 1]
```diff
@@ -1741,4 +1742,4 @@ API | Description | Auth | HTTPS | CORS |
 <br>
 
 ## License
-[MIT](LICENSE)
+[MIT](LICENSE)
```
Comment (neosergio) [inline README.md:1745]: IMHO you should not remove the newline at end of file

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @wisentini ;)

Can you resolve the review in https://github.com/public-apis/public-apis/pull/2889#discussion_r744294121 ?

File: README.md (hunk 1 above)
Comment (wisentini) [reply README.md:1745]: Oh, my bad! I didn't notice the newline had been removed, probably done by some setting on Visual Studio Code. I'll fix that asap!

File: README.md [hunk 2]
```diff
@@ -1338,6 +1338,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Open Notify](http://open-notify.org/Open-Notify-API/) | ISS astronauts, current location, etc | No | No | Unknown |
 | [Open Science Framework](https://developer.osf.io) | Repository and archive for study designs, research materials, data, manuscripts, etc | No | Yes | Unknown |
 | [Purple Air](https://www2.purpleair.com/) | Real Time Air Quality Monitoring | No | Yes | Unknown |
+| [Quadratic Solver](https://quadratic-solver-api.herokuapp.com/) | A Quadratic Solver | No | Yes | Unknown |
```
Comment (Carloscota7676) [inline README.md:1341]: No

**Review by Carloscota7676 — APPROVED** [review]

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ;)

Thanks for the update @wisentini

**Pull request merged**